coordinator/src/main/scala/com/socrata/datacoordinator/service/DynamicPortMap.scala

```diff
@@ -1,15 +1,17 @@
 package com.socrata.datacoordinator.service
 
-import com.typesafe.scalalogging.slf4j.Logging
+import org.slf4j.Logger
 
-trait DynamicPortMap extends Logging  {
+trait DynamicPortMap {
+
+  val log: Logger
```

Comment:
ok, i thought you are more concerned about the main class.  I'll just remove logger here.